README.md

```diff
@@ -1 +1,37 @@
-# kafka-broker-rack-awareness-operator
\ No newline at end of file
+# Kafka Broker rack awareness Operator
+
+Charmhub package name: Kafka Broker rack awareness
+More information: https://charmhub.io/kafka-broker-rack-awareness
+
+## Overview
+
+This charm handles the configuration of the `broker.rack` setting for Kafka. It needs to be used alongside the [Kafka operator](https://charmhub.io/kafka).
+
+## Usage
+
+### Basic usage
+
+The rack awareness Operator needs to be deployed to the same machine as the Kafka operator. First create a machine:
+```shell
+$ juju add-machine --series=jammy
+```
```

Comment:
I would also add that alternatively one could JUST deploy a kafka cluster (with two machines) and the deploy the kafka-broker-rack-awareness charm on those machines. 

I would only add the case here of having a multiple broker, where we deploy two kafka-broker-rack-awareness charm with different config value